### PR TITLE
Update flux bootstrap and env files

### PR DESCRIPTION
**Pull request by emirgens**

### CODE REVIEW

File: scripts/radix-zone/radix_zone_playground.env [hunk 1]
```diff
@@ -65,6 +65,7 @@ AZ_REDIS_CACHE_SKU="Standard"
 
 # Shared resources
 AZ_RESOURCE_KEYVAULT="radix-vault-${RADIX_ENVIRONMENT}"
+AZ_COMMON_KEYVAULT="radix-keyv-playground"
```
Comment (Richard87) [inline scripts/radix-zone/radix_zone_playground.env:68]:
```suggestion
AZ_COMMON_KEYVAULT="radix-keyv-${RADIX_ZONE}"
```

File: scripts/radix-zone/radix_zone_dev.env [hunk 2]
```diff
@@ -66,6 +66,7 @@ AZ_REDIS_CACHE_SKU="Basic"
 
 # Shared resources
 AZ_RESOURCE_KEYVAULT="radix-vault-${RADIX_ENVIRONMENT}"
+AZ_COMMON_KEYVAULT="radix-keyv-dev"
```
Comment (Richard87) [inline scripts/radix-zone/radix_zone_dev.env:69]:
```suggestion
AZ_COMMON_KEYVAULT="radix-keyv-${RADIX_ZONE}"
```

File: scripts/radix-zone/radix_zone_c2.env [hunk 3]
```diff
@@ -64,6 +64,7 @@ AZ_REDIS_CACHE_SKU="Standard"
 
 # Shared resources
 AZ_RESOURCE_KEYVAULT="radix-vault-${RADIX_ZONE}-${RADIX_ENVIRONMENT}"
+AZ_COMMON_KEYVAULT="radix-keyv-c2"
```
Comment (Richard87) [inline scripts/radix-zone/radix_zone_c2.env:67]:
```suggestion
AZ_COMMON_KEYVAULT="radix-keyv-${RADIX_ZONE}"
```

**Review by Richard87 — CHANGES_REQUESTED** [review]
We should change the hardcoded strings to follow the same convention, added some suggestions :)
ext-mon config is done in the radix-monitoring currently:
https://github.com/equinor/radix-monitoring/blob/master/cluster-external-monitoring/scripts/radix-zone/radix_zone_ext-mon.env

Maybe we should remove radix_zone_extmon.env to avoid confusion?

The rest looks good! 👍

**Review by sveinpj — APPROVED** [review]

**Pull request merged**